tests/integration_tests/functional/test_aarch64_armshaker.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Opcode fuzzer test for aarch64."""
+
+# pylint: disable=too-many-lines
+
+import platform
+import time
+
+import pytest
+
+from framework import utils
+from framework.utils_cpu_templates import nonci_on_arm
+
+PLATFORM = platform.machine()
+
+TIMEOUT = 4 * 60 * 60
```

Comment:
Will it always run for 4 hours? This will be a long and expensive test. Do we need to run it daily?

Reply:
On c7g it runs in ~100 mins, on m6g ~160 mins. This test can be run nightly.